src/cell.ts

```diff
@@ -0,0 +1,20 @@
+import interfaces = require('./interfaces');
+import IPy_Object = interfaces.IPy_Object;
+import enums = require('./enums');
+
+class Py_Cell implements IPy_Object {
+    ob_ref: IPy_Object;
+    
+    public constructor(ob_ref: IPy_Object){
+        this.ob_ref = ob_ref;
+    }
+    
+    public getType(): enums.Py_Type { return enums.Py_Type.OTHER; }
+    
+    public hash(): number {
+        return -1;
```

Comment:
At some point, these all need to be renamed to `__hash__` and we'll need to add tests for their behavior (not their values). Looks like the default implementations are [fairly straightforward](http://docs.sympy.org/dev/python-comparisons.html) for many types.